rakelib/pwsh.rake

```diff
@@ -219,6 +223,21 @@ namespace :pwsh do
             ruby_arg:                   'bare',
             validate_not_null_or_empty: true
           }
+        when 'module'
+          # bolt module install
+          # bolt module add [module]
+          mod_param_mandatory = (@pwsh_command[:verb] == 'Add')
```

Comment:
This will still result in `Get-BoltModule`, `Install-BoltModule`, and `Register-BoltModuleTypes` having an optional `Module` parameter, which is not correct:

```
function Register-BoltModuleTypes
{
  [CmdletBinding()]
  param (
    [Parameter(Position=0)]
    [ValidateNotNullOrEmpty()]
    [Alias('md')]
    [string]
    $Module,
```